drapps/README.md

```diff
@@ -1,110 +1,100 @@
+    -p, --path DIRECTORY  Path to folder with files that should be uploaded
+                        [required]
+    -n, --name TEXT       Name for new custom application. Default: CustomApp
+    -t, --token TEXT      Pubic API access token.
+    -E, --endpoint TEXT   Data Robot Public API endpoint
+    --help                Show this message and exit.
 
-UUID or name of Execution Environment that will be used as base for your app setup.
-Execution Environment contains preinstalled essential libraries and packages, that 
-can be used by your application.
+```
 
-You can find list of available Execution Environments in **Custom Model Workshop**
-on [**Environments page**](https://app.datarobot.com/model-registry/custom-environments)
+More detailed descriptions for each argument are provided in the table below:
 
-For a custom Streamlit application, use: `--base-env '[DataRobot] Python 3.9 Streamlit'`.
+Argument     | Description
+-------------|-------------
+`--base-env` | Enter the UUID or name of execution environment used as base for your Streamlit app. The execution environment contains the libraries and packages required by your application. You can find list of available environments in the **Custom Model Workshop** on the [**Environments**](https://app.datarobot.com/model-registry/custom-environments) page. <br> For a custom Streamlit application, use `--base-env '[DataRobot] Python 3.9 Streamlit'`.
+`--path`     | Enter the path to a folder used to create the custom application. Files from this folder are uploaded to DataRobot and used to create the custom application image. The custom application is started from this image. <br> To use the current working directory, use `--path .`.
+`--name`     | Enter the name of your custom application. This name is also used to generate the name of the custom application image, adding `Image` suffix. <br> The default value is `CustomApp`.
+`--token`    | Enter your API Key, found on the [**Developer Tools**](https://app.datarobot.com/account/developer-tools) page of your DataRobot account. <br> You can also provide your API Key using the `DATAROBOT_API_TOKEN` environment variable.
+`--endpoint` | Enter the URL for the DataRobot Public API. The default value is `https://app.datarobot.com/api/v2`. <br> You can also provide the URL to Public API using the `DATAROBOT_ENDPOINT` environment variable.
 
-### --path
+## Deploy an example app
 
-Path to folder which will be used to create custom application. Files from this folder
-will be uploaded to Data Robot and used for creating custom application image. Custom
-application will be started from this image.
+To test this, deploy an example Streamlit app using the following command from 
+the [`./drapps`](https://github.com/datarobot/datarobot-user-models/tree/master/drapps) directory:
 
-To use the current working directory use `--path .`.
+``` sh
+./drapps.py -t <your_api_token> -e "[Experimental] Python 3.9 Streamlit" -p ./demo-streamlit
```

Comment:
For the `demo-streamlit` directory, I would also recommend fixing the typo in the filename of `demo-streamlet.py` (and the associated command in `start-app.sh`).

Reply:
Oh, thank you.

Reply:
@demchukilya Also not sure if the `[Experimental]` tag  on the environment will still be there when the feature is GA, but I didn't want to make that kind of change in this review.

Reply:
I think we should keep Experimental at least for now.

Reply:
Sounds good! Feel free to merge this whenever! Not sure if we need more reviews.